include/deal.II/base/function.templates.h

```diff
@@ -26,11 +26,6 @@
 
 DEAL_II_NAMESPACE_OPEN
 
-
-template <int dim, typename Number>
-const unsigned int Function<dim, Number>::dimension;
```

Comment:
The *declaration* is there. Also the *initialization*. But this is the place where you *define* it, i.e., where the compiler allocates space for the variable.